platforms/web/styles/index.ts

```diff
@@ -56,7 +56,7 @@ export const GlobalStyle = createGlobalStyle<GlobalStyleProps>`
 
   body {
     color: ${({ theme, textColor }) =>
-      getColorShade(textColor ?? 'Shading.950', theme)};
+      getColorShade(textColor ?? 'Shading.900', theme)};
```

Comment:
Was this change intentional? I think it's unrelated to the PR.

Reply:
Not sure tbh, but I'll change it back.